python/py_src/sudachipy/sudachipy.pyi

```diff
@@ -26,12 +26,18 @@ class Dictionary:
     """
 
     @classmethod
-    def __init__(self, config_path: str = ..., resource_dir: str = ..., dict_type: str = None) -> None:
+    def __init__(cls, config_path: Optional[str] = ..., resource_dir: Optional[str] = ..., dict_type: Optional[str] = None) -> None:
         """
         Creates a sudachi dictionary.
 
         If both config.systemDict and dict_type are not given, `sudachidict_core` is used.
         If both config.systemDict and dict_type are given, dict_type is used.
+
+        :param config_path: path to the configuration JSON file
+        :param resource_dir: path to the resource directory folder
+        :param dict_type: type of pre-packaged dictionary, referring to sudachidict_<dict_type> packages on PyPI: https://pypi.org/search/?q=sudachidict.
```

Comment:
How about renaming this arg e.g. `dict_type_or_path`?

Reply:
Maybe just `dict`?

Reply:
It is also a problem that changing parameter name in python is a breaking change. Need to handle that situation.

```python
Dictionary(dict_type="...") # this will break if just rename
```